lib/vcs.ml

```diff
@@ -64,28 +64,29 @@ let find_git () =
       | dir, (_, `Exited 0) -> Ok (Some (v `Git git ~dir:Fpath.(v dir)))
       | _ -> Ok None )
 
-let cmd_error cmd = function
-  | `Exited c ->
-      R.error_msgf "The following command exited with code %d:@\n%a" c Cmd.pp
-        cmd
-  | `Signaled c ->
-      R.error_msgf "The following command exited with signal %d:@\n%a" c Cmd.pp
-        cmd
+module D = struct
```

Comment:
How about naming this module `Default` so it's unambiguous? Sorry I wanted to suggest that during the first review round but then forgot about it!

Reply:
Done. It was already called `D` before and I just left the name (and moved it up a bit since I need it in more places now), but I also like calling it `Default` better.